recipes/multimodal/server/unified_server.py

```
@@ -0,0 +1,448 @@
+            self.pending_requests = self.pending_requests[self.batch_size :]
+
+        try:
+            requests = [p.request for p in batch]
+
+            if DEBUG:
+                print(f"[Batcher] Processing batch of {len(requests)} requests")
+
+            loop = asyncio.get_event_loop()
+            results = await loop.run_in_executor(None, self.backend.generate, requests)
+
+            for pending, result in zip(batch, results):
```

Ruff (0.15.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)